org.eclipse.lsp4e/.project

```diff
@@ -30,10 +30,16 @@
 			<arguments>
 			</arguments>
 		</buildCommand>
+		<buildCommand>
```

Comment:
this looks unrelated to the ticket you are resolving. Could you move the change to a new PR so that it can be looked at independently?